annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -703,6 +705,20 @@ private TranscriptConsequenceSummary getCanonicalTranscript(VariantAnnotation gn
         }
     }
 
+    private String resolveDbSnpRs(VariantAnnotation gnResponse, MutationRecord mRecord) {
+        String dbSnpRs = null;
+        if (gnResponse.getColocatedVariants() != null && !gnResponse.getColocatedVariants().isEmpty()) {
+            for (ColocatedVariant cv : gnResponse.getColocatedVariants()) {
+                Matcher dbSnpRsIdMatcher = DBSNP_RSID_REGIX.matcher(cv.getDbSnpId());
+                if (dbSnpRsIdMatcher.find()) {
+                    dbSnpRs = dbSnpRsIdMatcher.group(1);
```

Comment:
Could we use the whole match? Or is there a reason we look for specifically the first group?

Reply:
@averyniceday updated to whole match